nvdaHelper/remote/winword.cpp

```diff
@@ -690,6 +694,272 @@ inline bool generateFootnoteEndnoteXML(IDispatch* pDispatchRange, wostringstream
 	return true;
 }
 
+std::experimental::optional<int> getPageBreakType(IDispatchPtr pDispatchRange ) {

```

Comment:
Should this be perhaps called getSectionBreakType or some such? as from my reading it only returns something meaningful if the page break character particularly straddles 2 sections?